ia/src/load.py

```
@@ -0,0 +1,269 @@
+    client.last_look = [item.strip() for item in cleaned_response.split(',')]
+
+
+def is_look_response(message):
+    content = message.strip().lstrip('[').rstrip(']')
+    items = [item.strip() for item in content.split(',')]
+
+    return len(items) != utils.INVENTORY_ITEMS_COUNT
+
+
+def handle_Broadcast(client, response) -> None:
+    if (response == "ok"):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)